ciapp/src/test/java/com/group7/ciapp/ProjectTest.java

```diff
@@ -0,0 +1,62 @@
+package com.group7.ciapp;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class ProjectTest {
+
+    private static Project project;
+    private static final String testUrl = "https://github.com/dd2480-2025-group7/ci";
+    private static final String testCommitHash = "f8fffc3f13557144a1c2563f7628139031dbe7b1";
+    private static final Long testCheckId = 1L;
+
+    @BeforeAll
+    static void setUp() {
+        project = new Project(testUrl, testCommitHash, testCheckId);
+    }
+
+    /**
+     * Clone the repo and check if the path is correct. The path should not be null
+     * and the path should exist.
+     */
+    @Test
+    void testCloneRepo_true() {
+        String path = project.cloneRepo();
+
+        boolean pathExists = path.contains("/ci-tests/" + testCheckId);
```

Comment:
Maybe check that the directory contains something as well, not just that the variable looks good